lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -68,42 +73,42 @@ class FeatureCard extends React.Component {
     }
   }
 
-  getCampaigns(data, page, cb) {
-    this.requests.push($.get(`${API_URL}campaigns?staff_pick=${data.staff_pick}&page=${page}`, result => {
-      let rawCampaigns = result.data;
+  checkCampaignsRemaining() {
+    // If we're almost out of campaigns let's get more
+    if (this.state.campaigns.length < 4) {
+      // First update the API page
+      this.setState({campaignPage: this.state.campaignPage + 1});
+      // Then grab campaigns for that page & update the state
+      this.getCampaigns(this.state.campaignPage, campaigns => {
+        // this.setState({
```

Comment:
I left a fee things still commented out / undone because it's a wip, didn't mean to ask for review yet!